server/views/temporary-accommodation/bookings/_editable.njk

```diff
@@ -0,0 +1,54 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "../components/ta-input/macro.njk" import taInput %}
+{% from "../components/ta-date-input/macro.njk" import taDateInput %}
+
+{{ taInput(
+    {
+      label: {
+        text: "What is the person's CRN",
```

Comment:
I dunno if it might be better from a user's POV if we ask for the person's CRN first, and then confirm it on that page? There might be a risk that this could be booked for the wrong person if the user can't confirm the person's details. Ignore me if this is coming in the future though!

Reply:
Also, there might be unforseen circumstances if the person is a limited access offender, though I might be getting too deep in the weeds!

Reply:
Yep, we're planning to pull that step out, this is intentionally getting a quick "let us create a booking" in